lib/Cake/Console/Command/TestShell.php

```diff
@@ -226,6 +226,8 @@ protected function _runnerOptions() {
 
 		if (!empty($params['no-colors'])) {
 			unset($params['no-colors'], $params['colors']);
+		} elseif (strtolower(substr(php_uname(), 0, 3)) === 'win') {
```

Comment:
I am using it now too and indeed it outputs colours correctly. Is it worth adding a note to documentation about Windows support for colours?